Added documentation for Celery Beat auto monitoring (#6526)
getsentry/sentry-docs: src/platform-includes/crons/setup/python.celery.mdx

First, set up your Celery beat schedule:

```python
# tasks.py
from celery import Celery
from celery.schedules import crontab

app = Celery('tasks', broker='...')
app.conf.beat_schedule = {
    'set-in-beat-schedule': {
        'task': 'tasks.tell_the_world',
        'schedule': crontab(hour='10', minute='15'),
        'args': ("Some important message!", ),
    },
}
```

Initialize Sentry in the `celeryd_init` or `beat_init` signal. Make sure to pass your Celery `app` variable to the `CeleryIntegration`:

```python
# tasks.py
from celery import signals

import sentry_sdk
from sentry_sdk.integrations.celery import CeleryIntegration

#@signals.beat_init.connect
@signals.celeryd_init.connect
def init_sentry(**kwargs):
    sentry_sdk.init(
        dsn='___PUBLIC_DSN___',
        integrations=[CeleryIntegration(celery_app=app)],  # 👈 pass Celery app.
        environment="local.dev.grace",
        release="v1.0",
    )
```

> This will now auto discover all the tasks in your Celery Beat schedule and will capture telemetry data when the task is started, when it finishes or fails.

Reviewer: ```suggestion
Once Sentry Crons is set up, tasks in your Celery Beat schedule will be auto-discoverable, and telemetry data will be captured when a task is started, when it finishes, and when it fails. 

```

Reviewer 2: I edited this suggestion to lowercase "beat". AFAICT the extension is called "Celery Beat" but it refers to its worker and beat services with lowercase.

> 
> <Note>
> 
> You **do not need to create Cron Monitors** for your tasks on Sentry.io, we will do this for you.
> 
> </Note>
> 
> Start your Celery Beat and workers and see your tasks being monitored at [https://sentry.io/crons/](https://sentry.io/crons/).
> 

Reviewer: ```suggestion
Start your Celery Beat and workers and see your tasks being monitored at [https://sentry.io/crons/](https://sentry.io/crons/).

<Note>

You don't need to create Cron Monitors for your tasks on Sentry.io, we'll do it for you.

</Note>

```
It makes for a smoother user experience to have the note at the end :)

Reviewer 2: edited your suggestion: "Celery Beat and workers" -> "Celery beat and worker services"

> ## Manual Task monitoring
> 
> We provide a lightweight decorator to make it easy to monitor individual tasks.
> 
> Add the `@sentry_sdk.monitor` decorator to your Celery task (or really just any function) and supply a `monitor_slug` of a monitor created previously on Sentry.io.
> Every time the task (or function) is now executed telemetry data will be captured when the task is started, when it finishes or fails.
> 
> <Note>
> 
> Make sure the Sentry `@sentry_sdk.monitor` decorator is **below** Celery's `@app.task` decorator.
> 
> </Note>

Reviewer: ```suggestion
## Manual Task Monitoring

We provide a lightweight decorator to make monitoring individual tasks easier. To use it, add `@sentry_sdk.monitor` to your Celery task (or any function), then supply a `monitor_slug` of a monitor created previously on Sentry.io. Once this is done, every time the task (or function) is executed, telemetry data will be captured when a task is started, when it finishes, and when it fails.

<Note>

Make sure the Sentry `@sentry_sdk.monitor` decorator is below Celery's `@app.task` decorator.

</Note>
```
